src/CPPScope.cxx

```diff
@@ -431,10 +431,7 @@ static PyObject* meta_getattro(PyObject* pyclass, PyObject* pyname)
         if (attr) {
         // cache the result
             if (CPPDataMember_Check(attr)) {
-                if (Cppyy::IsClass(scope))
-                    PyType_Type.tp_setattro(pyclass, pyname, attr);
-                else
-                  PyType_Type.tp_setattro((PyObject*)Py_TYPE(pyclass), pyname, attr);
+                PyType_Type.tp_setattro((PyObject*)Py_TYPE(pyclass), pyname, attr);
```

Comment:
Doing the same as [upstream](https://github.com/wlav/CPyCppyy/blob/faf033170dd1704ee76f015f8ab764e8a3197487/src/CPPScope.cxx#L440-L445).